src/Dialogs/MultiFilePicker.cpp

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The XCSoar Project
+
+#include "MultiFilePicker.hpp"
+#include "../Form/DataField/MultiFile.hpp"
+#include "Dialogs/FilePicker.hpp"
+#include "Form/DataField/MultiFile.hpp"
+#include "Language/Language.hpp"
+#include "ListPicker.hpp"
+#include "Look/DialogLook.hpp"
+#include "Renderer/TextRowRenderer.hpp"
+#include "UIGlobals.hpp"
+#include "Widget/TextWidget.hpp"
+#include "Widget/TwoWidgets.hpp"
+#include "WidgetDialog.hpp"
+
+static constexpr int mrRemove = 601;
+static constexpr int mrAdd = 600;
+
+class MultiFilePickerSupport : public ListItemRenderer {
+
+  TextRowRenderer row_renderer;
+  std::vector<Path> &active_files;
+
+public:
+  explicit MultiFilePickerSupport(std::vector<Path> &_active_files)
+      : active_files(_active_files)
+  {
+  }
```

Comment:
I know { } on new line. but since this construtctor is empty you can put them on the same line as the definiton.